apps/ledger-live-mobile/src/components/Nft/NftLink.tsx

```diff
@@ -0,0 +1,73 @@
+import { Flex, Box, Text, IconsLegacy, rgba } from "@ledgerhq/native-ui";
+import React, { FC } from "react";
+import { TouchableOpacity, View } from "react-native";
+import styled, { useTheme } from "styled-components/native";
+
+interface Props {
+  readonly style?: React.ComponentProps<typeof TouchableOpacity>["style"];
+  readonly leftIcon: React.ReactNode;
+  readonly rightIcon?: React.ReactNode;
+  readonly title: string;
+  readonly subtitle?: string;
+  readonly onPress?: React.ComponentProps<typeof TouchableOpacity>["onPress"];
+  readonly primary?: boolean;
+  readonly testID?: string;
+}
+
+const NftLink: FC<Props> = ({
```

Comment:
NOTE: This component already existed but has simply been moved to its own file so it can be reused in another drawer.